File: truezip-kernel/src/test/java/de/schlichtherle/truezip/kernel/ArchiveManagerTest.java
/*
 * Copyright (C) 2005-2012 Schlichtherle IT Services.
 * All rights reserved. Use is subject to license terms.
 */
package de.schlichtherle.truezip.kernel;

import de.truezip.kernel.FsManager;
import de.truezip.kernel.FsManagerTestSuite;
import de.truezip.kernel.util.Link.Type;

/**
 * @author Christian Schlichtherle
 */
public class ArchiveManagerTest extends FsManagerTestSuite {

    @Override
    protected FsManager newManager(Type type) {
        return new ArchiveManager(type);
    }
}

File: truevfs-kernel/truevfs-kernel-impl/src/test/scala/net/truevfs/kernel/impl/ArchiveManagerTest.scala
/*
 * Copyright (C) 2005-2012 Schlichtherle IT Services.
 * All rights reserved. Use is subject to license terms.
 */
package net.truevfs.kernel.impl

import net.truevfs.kernel.spec._
import net.truevfs.kernel.spec.util.Link._

/**
  * @author Christian Schlichtherle
  */
class ArchiveManagerTest extends FsManagerTestSuite {
  override def newManager(tµpe: Type): FsManager = new ArchiveManager(tµpe)
}
